packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v0.60.0
```

Comment:
Is this meant to be included in your PR? Looks like it could be a dodgey merge/rebase?